rasa_core/training/interactive.py

```diff
@@ -588,6 +588,17 @@ def _ask_if_quit(sender_id: Text, endpoint: EndpointConfig) -> bool:
     elif answer == "undo":
         raise UndoLastStep()
     elif answer == "fork":
+        # a fork is created and the first story line saved as if quit
+        story_path, nlu_path, domain_path = _request_export_info()
+
+        tracker = retrieve_tracker(endpoint, sender_id)
+        evts = tracker.get("events", [])
+
+        _write_stories_to_file(story_path, evts)
+        _write_nlu_to_file(nlu_path, evts)
+        _write_domain_to_file(domain_path, evts, endpoint)
+
+        logger.info("Successfully wrote stories and NLU data")
```

Comment:
since this whole chunk of code is used in two places now, I'd suggest we make a function and call that instead